lib/core_ext/active_record/connection_adapters/postgresql_adapter.rb

```diff
@@ -9,6 +9,12 @@ class PostgreSQLAdapter
       # Regex to find where clause in index statements
       INDEX_WHERE_EXPRESSION = /WHERE (.+)$/
 
+      # Taken from https://github.com/postgres/postgres/blob/master/src/include/catalog/pg_index.h#L75
+      # Values are in reverse order
+      INDOPTION_DESC        = 1
```

Comment:
Two questions:

1. Can "INOPTION" be renamed to "INDEX_OPTION"?
2. Any way to get those values by querying the db instead of defining constants?

Reply:
1 As I can see indoption column represents data as bit map. INDOPTION_DESC and INDOPTION_NULLS_FIRST constants contain bit mask to extract individual bits. So I think these two constants can also be named as INDOPTION_DESC_BITMAP and INDOPTION_NULLS_FIRST_BITMAP. But I took names from source files.
2. `d.indoption` in this query https://github.com/HornsAndHooves/pg_saurus/pull/27/files#diff-4a2082d4ef5f4eb897a2c07061b86676R87 can be changed with `(d.indoption & 1) = 1 as is_desc, (d.indoption & 2) = 2 as is_nulls_first`. In this case `is_desc` and `is_nulls_first` columns will indicate required order options.